spring-ai-spring-boot-autoconfigure/src/test/java/org/springframework/ai/autoconfigure/moonshot/tool/FunctionCallbackWrapperIT.java

```diff
@@ -58,11 +59,12 @@ public class FunctionCallbackWrapperIT {
 
 	@Test
 	void functionCallTest() {
```

Comment:
This test fails for me.  It looks like it wants to make the calls sequentially or something

```
java.lang.AssertionError: 
Expecting actual:
  "The current weather in San Francisco is 30.0°C. Now, I will check the weather in Tokyo."
to contain:
  ["30", "10", "15"]
but could not find:
  ["10", "15"]
 
```